format-delimited/src/main/java/io/cdap/plugin/format/delimited/common/DataTypeDetectorUtils.java

```diff
@@ -46,15 +46,15 @@ public class DataTypeDetectorUtils {
    */
   public static void detectDataTypeOfRowValues(Map<String, Schema> override,
                                                DataTypeDetectorStatusKeeper dataTypeDetectorStatusKeeper,
-                                               String[] columnNames, String[] rowValues) {
+                                               List<String> columnNames, String[] rowValues) {
```

Comment:
there should not be any changes in this class, it is unrelated to the path field.